src/client/components/annotations.ts

```diff
@@ -0,0 +1,157 @@
+import { LitElement, css, html, nothing } from 'lit'
+import { customElement, property } from 'lit/decorators.js'
+import { when } from 'lit/directives/when.js'
+
+import { ClientMessages } from '../../constants'
+import type { ClientMessage, NotebookCellAnnotations } from '../../types'
+import { getContext } from '../utils'
+import '@vscode/webview-ui-toolkit/dist/data-grid/index'
+
+type AnnotationsMutation = Partial<NotebookCellAnnotations>
+type AnnotationsKey = keyof NotebookCellAnnotations
+
+@customElement('edit-annotations')
+export class Annotations extends LitElement {
+  // Define scoped styles right with your component, in plain CSS
+  static styles = css`
+    :host {
+      display: block;
+      font-family: Arial;
+    }
+
+    section {
+      padding: 10px;
+      border: 1px solid #444;
+      border-radius: 5px;
+      display: flex;
```

Comment:
I don't think this style is affecting at all the annotations container children, it's only applying a flex style to the grid container ((vscode-data-grid) parent of which children are being affected by a grid display. (Just in case that style is having any specific intention)

Reply:
Would you mind taking care of this as part of working on the UI? I'm anything but a CSS ninja :-)